src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -47,6 +46,27 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Skip_analyzers_for_implicitly_triggered_builds}" />
                 </StackPanel>
             </GroupBox>
+            <GroupBox x:Uid="EditorHelpGroupBox"
```

Comment:
❔ Do we need to move this? Seems like we can keep it in the original location and make a separate decision to move.